eradiate/scenes/biosphere.py

```diff
@@ -45,287 +46,56 @@ class BiosphereFactory(BaseFactory):
     _constructed_type = SceneElement
     registry = {}
 
-
+@unit_enabled
 @attr.s
-class Canopy(SceneElement, ABC):
+class Tree(ABC):
```

Comment:
Well my first idea was, that all those plant-like objects would share one base type. But now that you mention it, I think it is not necessary, since the Canopy classes will be very specialized and I don't think I will be creating an interface, where the different `Canopy` classes can flexibly use different `tree` or `leaf cloud` objects. In other words: There is no reason to have on common base type that would be used for validation or similar.

Reply:
If we ever have to introduce a base type (_i.e._ if these objects eventually happen to have some common interface parts), we'll add it later.